app/uk/gov/hmrc/economiccrimelevyreturns/controllers/ContactRoleController.scala

```diff
@@ -36,12 +38,14 @@ class ContactRoleController @Inject() (
   val controllerComponents: MessagesControllerComponents,
   authorise: AuthorisedAction,
   getReturnData: DataRetrievalAction,
-  eclReturnsConnector: EclReturnsConnector,
+  eclReturnsService: ReturnsService,
   formProvider: ContactRoleFormProvider,
   pageNavigator: ContactRolePageNavigator,
   view: ContactRoleView
 )(implicit ec: ExecutionContext)
     extends FrontendBaseController
+    with BaseController
+    with ErrorHandler
     with I18nSupport {
 
```

Comment:
Sorry, I know that this is not where comment should be, you haven't changed that method so I didn't know where to put it :). In `onPageLoad` we use `contactName` from package object. Since we are moving towards implicit calls as much as possible, can me move logic from there to some implicit class either new one or `BaseController`. So we would have a call like `request.contactName` where `contactName` would be new method in implicit class. Also: this "old" `contactName` is throwing error that we are not handling.